topics/sequence-analysis/tutorials/annotation-with-prokka/slides.html

```diff
@@ -0,0 +1,41 @@
+---
+layout: introduction_slides
```

Comment:
Can you change layout to `tutorial_slides`? Thanks!

Reply:
Thanks, changed.